apps/src/templates/AgeDialog.jsx

```diff
@@ -88,8 +81,31 @@ class AgeDialog extends Component {
     }
   };
 
+  onClickAgeOk = () => {
+    const value = this.ageDropdown.getValue();
+    // Ignore click if nothing selected
+    if (!value) {
+      return;
+    }
+
+    // Sets cookie to true when anon user is 13+. False otherwise.
+    let over13 = parseInt(value, 10) >= 13;
+    this.setSessionStorage(over13);
+
+    if (over13) {
+      this.props.turnOffFilter();
+    }
+  };
+
   render() {
     const { signedIn} = this.props;
+
+    // If the song filter override has been turned on, set session storage
+    // Dialog won't render
+    if (queryString.parse(window.location.search).songfilter === 'on') {
+      this.setManualFilter();
```

Comment:
Will set sessionStorage.getItem(sessionStorageKey), so will return null in following condition.